frontend/src/features/mapSideBar/SidebarContents/BuildingForm.test.tsx

```diff
@@ -89,7 +89,7 @@ describe('Building Form', () => {
     await wait(() => {
       fireEvent.click(getByText(/Continue/i));
     });
-    expect(getByText(/Building Valuation/i)).toBeInTheDocument();
+    expect(getByText('Building Valuation', { exact: true })).toBeInTheDocument();
```

Comment:
changed this because there's text that says "building valuation" as well as the title, so this only gets the title

Reply:
I believe just taking out the `i` from Regex will also get you the desired behaviour. It sets the Regex to case insensitive so `/Building Valuation/` would be case-sensitive. But this way/ regex way both accomplish the same thing, just wanted to pass it on!